### ISSUE TITLE
Recieve error on all json files after setup

### ISSUE THREAD

**Issue by koerbcm** [Bkucera/eslint-plugin-json-format#12]
After installation and configuration, when I run an eslint command, all JSON files within the project now fail with the same error listed below. We have not experienced this issue before and Googling for a solution or a way to troubleshoot turned up no real results.  

Wondering if anyone else may have experienced this or if someone might have an idea as to how to troubleshoot the issue. `--debug` didn't seem to give me much more info and from all accounts, it seems like there wasn't an issue. 

**Error message:**
`1:2  error  Unexpected end of JSON input`

**Command Executed:**
```bash
eslint . --ext .json --fix
```

**.eslintrc**
```json
{
  "extends": ["@xxxxxxxxxx/eslint-config-prettier", "plugin:json/recommended", "prettier"],
  "env": {
    "es6": true,
    "node": true
  },
  "plugins": [
    "json-format",
    "json",
    "prettier"
  ],
  "parserOptions": {
    "ecmaVersion": 2018,
    "sourceType": "module"
  },
  "rules": {
    "prettier/prettier": "error",
    "max-len": ["error", {
      "code": 120,
      "comments": 200,
      "ignoreUrls": true,
      "ignoreTrailingComments": true,
      "ignoreStrings": true
    }]
  }
}
```

devDependencies:
```json
"devDependencies": {
    "@xxxxxxxxxxx/eslint-config-prettier": "^1.0.2",
    "bluebird": "3.7.2",
    "chai": "4.2.0",
    "chai-as-promised": "7.1.1",
    "chai-date-string": "^0.1.0",
    "eslint": "6.8.0",
    "eslint-plugin-import": "2.19.1",
    "eslint-plugin-json": "2.0.1",
    "eslint-plugin-json-format": "^2.0.1",
    "eslint-plugin-mocha": "6.2.2",
    "eslint-plugin-prettier": "^3.1.2",
    "gh-pages-publish": "0.0.5",
    "gitbook-cli": "2.3.2",
    "mocha": "7.0.1",
    "mocha-multi-reporters": "1.1.7",
    "nock": "11.7.1",
    "nodemon": "2.0.2",
    "nyc": "15.0.0",
    "qart": "2.1.2",
    "rewire": "4.0.1",
    "sinon-chai": "3.4.0"
  },
```

**Debug output:**
```
eslint:ignore-pattern Check { filePath:
   '/xxxx/xxxx/xxxx/xxxx/config/dev.json',
  dot: false,
  relativePath: 'config/dev.json',
  result: false } +5ms
  eslint:file-enumerator Yield: dev.json +5ms
  eslint:cascading-config-array-factory Load config files for /xxxx/xxxx/xxxx/xxxx/config. +5ms
  eslint:cascading-config-array-factory Cache hit: /xxxx/xxxx/xxxx/xxxx/config. +0ms
  eslint:cli-engine Lint /xxxx/xxxx/xxxx/xxxx/config/dev.json +6ms
  eslint:linter Linting code for /xxxx/xxxx/xxxx/xxxx/config/dev.json (pass 1) +2ms
  eslint:linter Verify +0ms
  eslint:linter With ConfigArray: /xxxx/xxxx/xxxx/xxxx/config/dev.json +0ms
  eslint:linter Apply the processor: 'json/.json' +0ms
  eslint:linter A code block was found: '(unnamed)' +16ms
  eslint:linter Generating fixed text for /xxxx/xxxx/xxxx/xxxx/config/dev.json (pass 1) +3ms
  eslint:source-code-fixer Applying fixes +21ms
  eslint:source-code-fixer No fixes to apply +0ms
```

**Comment by Bkucera** [comment]
@koerbcm Why do you have `eslint-plugin-json` as well as this plugin? That's probably the issue

**Comment by koerbcm** [comment]
I noticed that this one doesn't have a lot of contextual feedback when there are issues but it formats the json and dotfiles quite well. Also `eslint-plugin-json` has some options that we can set. However, I will try to remove it and see what happens.

**Comment by koerbcm** [comment]
It does look like having both was the issue and while indention errors are hard to parse, other issues within the JSON seem to be formatted a lot better more readable. Thank you for pointing me in this direction.

**Comment by Bkucera** [comment]
@koerbcm I'll look into improving the errors. Thanks for the feedback

**Comment by pkuczynski** [comment]
I had an issue with duplicated keys in JSON files. The plugin `json` reported it nicely, but `json-format` not. Shame we can't use them both :(
